### ISSUE TITLE
add documentation for adding terms

### ISSUE THREAD

**Issue by wdduncan** [OBOFoundry/COB#203]
We need documentation (and/or policies) on how to add new terms to COB. We have a ton of experience to draw on (e.g., GO, OBI, RO, etc.). So, much of the work, may be a matter of deciding which approach(s) to adopt.

cc @cmungall @bpeters42 @matentzn

**Comment by matentzn** [comment]
Yeah, it would be nice to get a step by step process on a docs page on how to add a new term into COB. Simple list suffices, but some clarity on what is needed would be good.

**Comment by matentzn** [comment]
Can we elevate the urgency - we want to deal with #192 but I don't know which files need to be edited! Thank you!

@cmungall if you add a few bullets here, i can transform them into a docs page.

**Comment by matentzn** [comment]
I will make a temporary SOP for now:

1. Create an issue suggesting the new term. If it does not get any comments, chase the issue after two weeks.
2. If after four weeks no feedback is given, you can consider the term request non-controversial and proceed to make a pull request
3. If you don't yet have an ID range yet, make a pull request to [obtain one](https://github.com/OBOFoundry/COB/blob/master/src/ontology/cob-idranges.owl).
4. Using Protege, edit the src/ontology/cob-edit.owl file (editors file), and add the proposed class in the usual way.
5. If your class maps to an existing top level class in an OBO ontology, add your mapping here: https://github.com/OBOFoundry/COB/blob/master/src/ontology/components/cob-to-external.tsv
6. Once every 7 days, we should chase the pull request to be reviewed. 
7. The pull request can be merged by anyone with write access without a review one 3 months have passed without a review.

**Comment by wdduncan** [comment]
@matentzn Thanks for putting this together!
Some thoughts:
- Require more than one reviewer for acceptance (I'm thinking it might help improve quality).
- Perhaps establish a group for approving terms? Or maybe discussions of new terms could be part of obo-operations?
- Not sure we should ever allow a PR to merged w/o review. If a new terms hasn't been reviewed in 3 months, then there may be some other issue preventing review (e.g., lack of time to review, lack of interest in the term).

**Comment by matentzn** [comment]
You are probably right!
